### DISCUSSION TITLE
11/21 조지연 블로그 1회차

### DISCUSSION THREAD

**Discussion by oratio100** [Orm-camp/blog-feedback#37] [동글동글 3기 피드백]
[블로그 링크](https://velog.io/@nemonemo/Git-%EC%BB%A4%EB%B0%8B-%EC%88%9C%EC%84%9C-%EB%B3%80%EA%B2%BD%ED%95%98%EA%B8%B0git-rebase-i)

간단하게 작성한 블로그 글입니다.
편하게 피드백 남겨주세요~

**Comment by mh0223** [comment]
소제목들도 진한 글씨나 조금 더 큰 글씨로 강조하는 것도 좋을 것 같아요 !

**Comment by kimjigyun** [comment]
저는 길지 않은 글이어서 정리하신게 눈에 잘 보여서 좋았어요 ㅎㅎ

**Comment by Orm-camp** [comment]
저는 지연님 글에서 썸네일이 가장 눈에 띄었어요!
군더더기 없고 블로그 메인에서도 글을 찾기 쉽게 만들어 두신 게 정말 신의 한 수가 아닐까 싶습니다 👍🏻 
약간 블로그 2회차 하시는 것 같아요 ㅎㅎ

문단 간 구분이 더 확실하게 되어 있다면 좋을 것 같네요!
다음 글도 기대하겠습니다 😆

**Comment by ksj0621** [comment]
정말 제가 평소에 구글링 해서 정보를 얻는 능숙한 개발 블로거의 글 같았어요! 아직은 저는 지연님과 같은 고민을 해보지 않았지만 그런 순간이 온다면 지연님글을 알고리즘 타고 접할 것 같은 느낌ㅎㅎ
